client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -279,6 +280,7 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
 										disputeAcceptAction.acceptButtonTracksEvent,
 										{
 											dispute_status: dispute.status,
+											on_page: 'transaction_details',
```

Comment:
@brucealdridge Also for this event prop: the modal and "view" events are new to this screen, so this event prop may be redundant.

Reply:
I've created an issue for this #7474.